### PR TITLE
Update language for payments

### PR DESCRIPTION
**Pull request by sugargoat**

### CODE REVIEW

File: testnet-client/src/main.rs [hunk 1]
```diff
@@ -402,7 +402,7 @@ string that we send you. It should look something like:
                         u64_to_mob_display(fee),
                     );
                     println!(
-                        "balance after paying this bill will be {}.",
+                        "balance after sending this payment will be {}.",
```
Comment (eranrund) [inline testnet-client/src/main.rs:405]: While you're at it, I think it should be capital B in `Balance`?

Comment (rjwalters) [reply testnet-client/src/main.rs:405]: no, because it continues the line above; ideally we would test for 80 char width though.

**Comment by rjwalters** [comment]
Let's go through and scrub any remaining "bill" references before merging.

File: testnet-client/src/main.rs (hunk 1 above)
Comment (eranrund) [reply testnet-client/src/main.rs:405]: ah, missed that.

**Pull request merged**